whipper/command/cd.py

```diff
@@ -341,7 +350,19 @@ def doCommand(self):
         else:
             logger.info("creating output directory %s", dirname)
             os.makedirs(dirname)
-
+        self.coverArtPath = None
+        if (self.options.fetch_cover_art in {"embed", "complete"} and
```

Comment:
Why use a dict to compare against? It would likely be more sensible to use a tuple.
```suggestion
        if (self.options.fetch_cover_art in ("embed", "complete") and
```

I can cite you a number of arguments for why one would use tuple over list for something like this, but I don’t think anyone has every argued that using a dict would be a good idea.

Reply:
It isn't a `dict`, it's a `set`:

```python
>>> type({'embed', 'complete'})
<class 'set'>

Reply:
Indeed. Set literals are still new to me. :joy: